packages/uxpin-merge-cli/src/resources/Button/Button.js

```diff
@@ -18,17 +15,11 @@ function Button({ onClick, className, label, name, value, disabled }) {
 
 Button.propTypes = {
   onClick: PropTypes.func,
-  className: PropTypes.string,
-  name: PropTypes.string,
-  label: PropTypes.node,
-  value: PropTypes.string,
   disabled: PropTypes.bool,
+  label: PropTypes.node,
 };
 
 Button.defaultProps = {
```

Comment:
On second thoughts, let's remove `defaultProps`...I want to avoid people getting confused about using this or preset files (the preferred way)
```suggestion
```